sources/specUtils.ts

```diff
@@ -96,10 +100,15 @@ export async function loadSpec(initialCwd: string): Promise<LoadSpecResult> {
       continue;
 
     const manifestPath = path.join(currCwd, `package.json`);
-    if (!fs.existsSync(manifestPath))
-      continue;
+    try {
+      file = await fs.promises.open(manifestPath, `r`);
+    } catch (err) {
+      if ((err as NodeError)?.code === `ENOENT`) continue;
+      throw err;
+    }
 
-    const content = await fs.promises.readFile(manifestPath, `utf8`);
+    const content = await file.readFile(`utf8`);
+    await file.close();
```

Comment:
Can't we just use `readFile` and assign `{}` to `data` wen catching `ENOENT`?

Reply:
Sure, the only anti-pattern is to rely on `existsSync`, as its result might be outdated by the time the `readFile` calls is processed.